dependencies.gradle

```
@@ -2,4 +2,5 @@
 
 dependencies {
     api("com.github.GTNewHorizons:NotEnoughItems:2.8.0-GTNH:dev")
+    api('com.github.GTNewHorizons:GT5-Unofficial:5.09.51.440:dev')
```

Can you change this to a `compileOnly` and a `runtimeOnlyNonPublishable`? `api` means that all mods that depend on translocators will get gt5u pulled into their deps. `devOnlyNonPublishable` would also work, but you can't easily comment out the `runtimeOnlyNonPublishable` portion so I like to separate them.


```suggestion
    compileOnly('com.github.GTNewHorizons:GT5-Unofficial:5.09.51.440:dev')
    runtimeOnlyNonPublishable('com.github.GTNewHorizons:GT5-Unofficial:5.09.51.440:dev')
```